connector/src/test/scala/com/microsoft/kusto/spark/KustoSinkBatchE2E.scala

```diff
@@ -9,16 +9,16 @@ import com.microsoft.kusto.spark.KustoTestUtils.getSystemTestOptions
 import com.microsoft.kusto.spark.datasink.{KustoSinkOptions, SinkTableCreationMode}
```

Comment:
Currently this test class does not run 
every method tagged with "taggedAs KustoE2E " is ignored